allennlp/predictors/text_classifier.py

```diff
@@ -0,0 +1,30 @@
+from overrides import overrides
+
+from allennlp.common.util import JsonDict
+from allennlp.data import DatasetReader, Instance
+from allennlp.models import Model
+from allennlp.predictors.predictor import Predictor
+
+
+@Predictor.register('text_classifier')
+class TextClassifierPredictor(Predictor):
+    """
+    Predictor for any model that takes in a sentence and returns
+    a single class for it.  In particular, it can be used with
+    the :class:`~allennlp.models.basic_classifier.BasicClassifier` model
+    """
+    def __init__(self, model: Model, dataset_reader: DatasetReader) -> None:
```

Comment:
You can remove this `__init__`, because it's already implemented in the super class and you aren't adding anything :)

Reply:
Thanks, removing the `__init__` and the related imports, it was also being tagged by pylint.